Rubberduck.VBEEditor/Selection.cs

```diff
@@ -141,16 +144,30 @@ public override string ToString()
 
         public static bool operator >(Selection selection1, Selection selection2)
```

Comment:
Just an idea - maybe add an xml docs to the operator. That might light up in intellisense and help bring the attention to somehow strange implementation?

Reply:
Done